modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -18,10 +16,12 @@ import {getFrameState} from './get-frame-state';
 registerLoaders([Tile3DLoader, Tileset3DLoader, GLTFLoader]);
 
 const defaultProps = {
+  opacity: 1.0,
+  pickable: true,
```

Comment:
Why is `pickable` default on? Add a comment?

Reply:
@ibgreen is there any reason default `pickable` to true?

Reply:
No. I was also wondering why it was there.